### ISSUE TITLE
How would I add clustering to this example?

### ISSUE THREAD

**Issue by ubergeekNZ** [maptiler/angular-template-maplibre-gl-js#2]
I am trying to add this example https://maplibre.org/maplibre-gl-js-docs/example/cluster/ but struggling with it. Is it possible to provide an example?

**Comment by bolollo** [comment]
Hello, I will try to prepare an example. Are you using the ngx-maplibre-gl library (https://maplibre.org/ngx-maplibre-gl/)? If you are using the code from this repo, add the sample code to the map component file (map.component.ts)

**Comment by ubergeekNZ** [comment]
Yes I am using the example code. I want to implement a similar the mapbox example using clustering. 

https://visgl.github.io/react-map-gl/examples/clusters

**Comment by bolollo** [comment]
Hello, for Angular, you have a component similar to that of React, where you also have a cluster example. https://maplibre.org/ngx-maplibre-gl/demo/cluster.

Here is a template to start using MapTiler maps https://github.com/maptiler/get-started-angular-maplibre-gl-js.
